examples/footer/index.html

```diff
@@ -8,59 +8,85 @@
-          </li>
-          <li>
-            <a target="_blank" rel="noopener noreferrer" href="https://www.stjude.org/legal/notice-of-privacy-practices.html" title="Notice of Privacy Practices (HIPAA)">Notice of Privacy Practices (HIPAA)</a>
-          </li>
-          <li>
-            <a target="_blank" rel="noopener noreferrer" href="https://www.stjude.org/legal/notice-of-non-discrimination.html" title="Notice of Non-Discrimination">Notice of Non-Discrimination</a>
-          </li>
-        </ul>
-        <ul class="social-media">
-          <li>
-            <a target="_blank" rel="noopener noreferrer" href="https://www.facebook.com/stjuderesearch" class="fab fa-facebook-f">
-              <span class="sr-only">Facebook</span>
+    <footer>
+      <div class="container">
+        <div class="row">
+          <div class="col-6 col-lg-3 col-md-4 offset-xl-0 offset-lg-5 offset-md-4 offset-md-3 offset-3 footer-logo">
+            <a class="navbar-brand" href="https://stjude.cloud">
+              <span class="logo"></span>
             </a>
-          </li>
-          <li>
-            <a target="_blank" rel="noopener noreferrer" href="https://twitter.com/StJudeResearch" class="fab fa-twitter">
-              <span class="sr-only">Twitter</span>
-            </a>
-          </li>
-          <li>
-            <a target="_blank" rel="noopener noreferrer" href="https://www.instagram.com/stjuderesearch/" class="fab fa-instagram">
-              <span class="sr-only">Instagram</span>
-            </a>
-          </li>
-          <li>
-            <a target="_blank" rel="noopener noreferrer" href="https://www.linkedin.com/company/st-jude-childrens-research-hospital/" class="fab fa-linkedin">
-              <span class="sr-only">LinkedIn</span>
-            </a>
-          </li>
-          <li>
-            <a target="_blank" rel="noopener noreferrer" href="https://www.youtube.com/user/MyStJude" class="fab fa-youtube">
-              <span class="sr-only">YouTube</span>
-            </a>
-          </li>
-        </ul>
+            <a class="sjc-title" href="https://stjude.cloud">St. Jude Cloud</a>
+          </div>
+          <div class="col-md-3 col-6 offset-xl-0 offset-md-2 footer-about">
+            <h6>About St. Jude</h6>
+            <ul>
+              <li>
+                <a href="https://www.stjude.org">Home</a>
+              </li>
+              <li>
+                <a href="https://www.stjude.org/about-st-jude.html">About</a>
+              </li>
+              <li>
+                <a href="https://stjude.org/contact-us.html">Contact</a>
+              </li>
+            </ul>
+          </div>
+          <div class="col-md-3 col-6 offset-xl-0 offset-md-2 footer-apps">
+            <h6>St. Jude Cloud Apps</h6>
+            <ul>
+              <li>
+                <a href="https://cstn.stjude.cloud">CSTN</a>
+              </li>
+              <li>
+                <a href="https://platform.stjude.cloud">Platform</a>
```

Comment:
```suggestion
                <a href="https://platform.stjude.cloud">Genomics Platform</a>
```

Reply:
I would change the order to what will match in our blog post:

* Genomics Platform
* PeCan
* Visualization Community